src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/MinMax_PowerFxV1CompatibilityRules.txt

```diff
@@ -0,0 +1,346 @@
+
+>> Max(1/0)
+Error({Kind:ErrorKind.Div0})
+
+>> Min(1/0)
+Error({Kind:ErrorKind.Div0})
+
+>> Max(Time(3,4,5), Time(2,1/0,8))
+Error({Kind:ErrorKind.Div0})
+
+>> Min(Time(3,4,5), Time(2,1/0,8))
+Error({Kind:ErrorKind.Div0})
+
+>> Max(1, 3, Sqrt(-1), -3)
+Error({Kind:ErrorKind.Numeric})
+
+>> Min(1, 3, Sqrt(-1), -3)
+Error({Kind:ErrorKind.Numeric})
+
+// ****** Table + expression, numeric ******
+
+>> Max([1, 2, 3, 4, 5], Value)
+5
+
+>> Min([1, 2, 3, 4, 5], Value)
+1
+
+>> Max([1, 2, 3], 4)
+4
+
+>> Min([1, 2, 3], 4)
+4
+
+>> Max(Table({Name:"John Doe", DOB:Date(1985,3,5), Thing: "1"},{Name:"Jane Roe", DOB:Date(1986,10,19), Thing: "2"}),Thing)
+2
+
+>> Min(Table({Name:"John Doe", DOB:Date(1985,3,5), Thing: "1"},{Name:"Jane Roe", DOB:Date(1986,10,19), Thing: "2"}),Thing)
+1
+
+// ****** Table + expression, date/time ******
+
+>> Max(Table({Name:"John Doe", DOB:Date(1985,3,5)},{Name:"Jane Roe", DOB:Date(1986,10,19)}),DOB)
+Date(1986,10,19)
+
+>> Min(Table({Name:"John Doe", DOB:Date(1985,3,5)},{Name:"Jane Roe", DOB:Date(1986,10,19)}),DOB)
+Date(1985,3,5)
+
+>> Max(Sequence(10), Date(2000,1,10 + Value))
+Date(2000,1,20)
+
+>> Min(Sequence(10), Date(2000,1,10 + Value))
+Date(2000,1,11)
+
+// ****** Errors, table + expression ******
+
+>> Max([1, 2, 1/0, 4, 5], Value)
+Error({Kind:ErrorKind.Div0})
+
+>> Min([1, 2, 1/0, 4, 5], Value)
+Error({Kind:ErrorKind.Div0})
+
+>> Max(If(Sqrt(-1) < 1, [1,2,3]), Value)
+Error({Kind:ErrorKind.Numeric})
+
+>> Min(If(Sqrt(-1) < 1, [1,2,3]), Value)
+Error({Kind:ErrorKind.Numeric})
+
+// ****** Compiler errors ******
+
+>> Max(1, 2, {a:3})
+Errors: Error 10-15: Expected number. We expect a number at this point in the formula.
+
+>> Min(1, 2, {a:3})
+Errors: Error 10-15: Expected number. We expect a number at this point in the formula.
+
+>> Max({a:1}, {a:2}, {a:3})
+Errors: Error 4-9: Expected number. We expect a number at this point in the formula.|Error 11-16: Expected number. We expect a number at this point in the formula.|Error 18-23: Expected number. We expect a number at this point in the formula.
+
+>> Min({a:1}, {a:2}, {a:3})
+Errors: Error 4-9: Expected number. We expect a number at this point in the formula.|Error 11-16: Expected number. We expect a number at this point in the formula.|Error 18-23: Expected number. We expect a number at this point in the formula.
+
+>> Min([1,2,3], [1])
+Errors: Error 13-16: Expected number. We expect a number at this point in the formula.|Error 0-17: The function 'Min' has some invalid arguments.
+
+>> Max([1,2,3], [1])
+Errors: Error 13-16: Expected number. We expect a number at this point in the formula.|Error 0-17: The function 'Max' has some invalid arguments.
+
+>> Min(Date(1985,3,5), DateTime(2014,11,11,13,50,0), Time(12, 1, 7))
+Date(1899,12,30)
+
+>> Min(DateTime(2014,11,11,13,50,0), Date(1985,3,5), Time(12, 1, 7))
+DateTime(1899,12,30,12,1,7,0)
+
+>> Min(Time(12, 1, 7), Date(1985,3,5), DateTime(2014,11,11,13,50,0))
+Time(0,0,0,0)
```

Comment:
This should be `Time(12,1,7,0)`

Reply:
As discussed, Date(1985,3,5) will be coerced to Time(0,0,0,0), hence this result